### PR TITLE
Site Editor: Make resizable frame compatible with RTL languages

### PR DESCRIPTION
**Pull request by t-hamano**
## What?
This PR ensures that the resize handles in the site editor are positioned correctly in RTL languages.

## How?

I first considered replacing `left` with `inset-inline-start`, which is a logical property, but `framer-motion` doesn't seem to support this property: https://github.com/framer/motion/issues/1989

Therefore, I switched between `left` and `right` depending on the language direction.

## Testing Instructions

- Switch to an RTL locale and access the site editor.
- Press the Tab key to navigate to the resize handle.
- The resize handle should appear between the sidebar navigation and the editor canvas.
- Confirm that you can resize the editor canvas with the left and right keys.
- This should work as usual in LTR languages.

### Before

before

### After

after

### CODE REVIEW

File: packages/edit-site/src/components/resizable-frame/index.js [hunk 1]
```diff
@@ -171,7 +171,12 @@ function ResizableFrame( {
 		event.preventDefault();
 
 		const step = 20 * ( event.shiftKey ? 5 : 1 );
-		const delta = step * ( event.key === 'ArrowLeft' ? 1 : -1 );
+		const delta =
+			step *
+			( ( ! isRTL() && event.key === 'ArrowLeft' ) ||
+			( isRTL() && event.key === 'ArrowRight' )
+				? 1
+				: -1 );
```
Comment (stokesman) [inline packages/edit-site/src/components/resizable-frame/index.js]: I’d be tempted to use multiplication and flip the delta again just in hopes it reads better but it’s fine as is.
```suggestion
			( event.key === 'ArrowLeft' ? 1 : -1 ) *
			( isRTL() ? -1 : 1 );
```

**Review by stokesman — COMMENTED** [review]
Nice to see fixes before an issue is even reported. It’ll be like it never happened 😉 but then again I didn’t search for an issue—maybe there is one.

The keyboard resizing works just fine. It happens that I tried dragging and the axis is flipped.

https://github.com/user-attachments/assets/7e57cfed-599f-44a7-965b-861e1e494f1f

That may as well be addressed here too. Here’s a change I tried that seemed to be the minimal way to get it working as expected:

```diff
diff --git a/packages/edit-site/src/components/resizable-frame/index.js b/packages/edit-site/src/components/resizable-frame/index.js
index 0585a3182d..d90aa98601 100644
--- a/packages/edit-site/src/components/resizable-frame/index.js
+++ b/packages/edit-site/src/components/resizable-frame/index.js
@@ -253,10 +253,11 @@ function ResizableFrame( {
 			size={ frameSize }
 			enable={ {
 				top: false,
-				right: false,
 				bottom: false,
 				// Resizing will be disabled until the editor content is loaded.
-				left: isReady,
+				...( isRTL()
+					? { right: isReady, left: false }
+					: { left: isReady, right: false } ),
 				topRight: false,
 				bottomRight: false,
 				bottomLeft: false,
@@ -276,7 +277,7 @@ function ResizableFrame( {
 			onMouseOver={ () => setShouldShowHandle( true ) }
 			onMouseOut={ () => setShouldShowHandle( false ) }
 			handleComponent={ {
-				left: canvasMode === 'view' && (
+				[ isRTL() ? 'right' : 'left' ]: canvasMode === 'view' && (
 					<>
 						<Tooltip text={ __( 'Drag to resize' ) }>
 							{ /* Disable reason: role="separator" does in fact support aria-valuenow */ }

```

**Comment by t-hamano** [comment]
> It happens that I tried dragging and the axis is flipped.

Good catch! I fixed it using the code you suggested.

**Review by stokesman — APPROVED** [review]
Thanks for the update! Retested and everything works as expected.

I assume the e2e test failures on the latest commits were errant.

**Comment by t-hamano** [comment]
Thanks for the review!

Regarding the E2E test failure, the cause is being explored here: https://wordpress.slack.com/archives/C02QB2JS7/p1727152145938689

I would like to merge this once this issue is resolved.

**Pull request merged**